Helper/Order.php

```diff
@@ -1388,4 +1391,16 @@ protected function validateCaptureAmount(OrderInterface $order, $captureAmount)
             throw new \Exception( __('Capture amount is invalid'));
         }
     }
+
+    /**
+     * @param OrderInterface $order
+     * @param $transactionState
+     * @return bool
+     */
+    protected function getPaymentUpdateFromAdminPanel(OrderInterface $order, $transactionState)
```

Comment:
While this method is doing a little more than what the name is advertising by checking order state and transaction state, and generally, I would suggest that a method returning a boolean by convention would have the verb "to be" usually as "is" somewhere in it (e.g. `isAnAllowedUpdateFromAdminPanel`), the code looks functionally good to me.